validate/validate.go

```diff
@@ -550,6 +553,9 @@ func validatePlatformInfo(platformInfo uint64, required *abi.SnpPlatformInfo) er
 	if !reportInfo.AliasCheckComplete && required.AliasCheckComplete {
 		return errors.New("required memory alias check hasn't been completed")
 	}
+	if !reportInfo.TIOEnabled && required.TIOEnabled {
+		return errors.New("required platform feature SEV-TIO not enabled")
```

Comment:
Intuitively, I would have done it the other way round (unauthorized platform feature). Isn't TIO extending the TCB to devices, and if disabled, the devices just won't be connected to the CVM?

Reply:
Maybe that's the case, maybe it's not. I can't find anything in the spec about that.

Reply:
It's described in 2.2 in https://www.amd.com/content/dam/amd/en/documents/epyc-technical-docs/specifications/58271.pdf